dev/io.openliberty.jakarta.jsonb.internal_fat/fat/src/io/openliberty/jakarta/jsonb/internal/fat/YassonTranslationTest.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.jakarta.jsonb.internal.fat;
+
+import java.util.Map;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.MinimumJavaLevel;
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import test.yasson.translation.web.YassonTranslationTestServlet;
+
+@MinimumJavaLevel(javaLevel = 11)
+@RunWith(FATRunner.class)
+public class YassonTranslationTest extends FATServletClient {
+
+    private static final String APP_NAME = "yassontranslationtestapp";
+    private static final String CONTEXT = "YassonTranslationTestServlet";
+
+    @Server("io.openliberty.jakarta.yasson.internal.fat.translation")
+    @TestServlet(servlet = YassonTranslationTestServlet.class, contextRoot = CONTEXT)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ShrinkHelper.defaultApp(server, APP_NAME, "test.yasson.translation.web");
+    }
+
+    @Test
+    public void testYassonTransaltionFunction() throws Exception {
+        // Ensure default server settings
+        Map<String, String> jvmOptions = server.getJvmOptionsAsMap();
+        jvmOptions.remove("-Duser.language");
+        jvmOptions.remove("-Duser.country");
+        server.setJvmOptions(jvmOptions);
+
+        server.startServer();
+
+        runTest(server, APP_NAME + "/" + CONTEXT, "testTranslationMessageDefault");
+        runTest(server, APP_NAME + "/" + CONTEXT, "testTranslationMessageProvidedLocale");
+
+        server.stopServer();
```

Comment:
If the test fails, won't we leak the server running?  Should the server.stopServer() either be moved to a @After method using if isServerStarted() check before calling stopServer() or just have that additionally to what we are doing here.  We could also change to do a try / finally instead around the runtTest method invocations.?

Reply:
Fixed in latest commit.